### ISSUE TITLE
Incorrect Texture Mapping

### ISSUE THREAD

**Issue by apptaro** [KhronosGroup/COLLADA2GLTF#199]
Some of converted GLTF files have incorrect texture mappings:
<img width="274" alt="summary" src="https://user-images.githubusercontent.com/998626/42938616-cf4d8bfe-8b8d-11e8-9c2a-9a49673876d7.png">

**Comment by apptaro** [comment]
Version: COLLADA2GLTF-v2.1.2-windows-Release-x64

DAE and GLTF data attached:
[data-uv.zip](https://github.com/KhronosGroup/COLLADA2GLTF/files/2209770/data-uv.zip)

Possibly related issue: #174, #175, #116

**Comment by apptaro** [comment]
All DAE files are exported from 3ds Max.
BabylonJS, Three.js and Cesium all render in the same way.

**Comment by lasalvavida** [comment]
This should be mostly fixed in the latest version of #180 that allows for multiple texture maps.

![screenshot from 2018-07-20 08 29 01](https://user-images.githubusercontent.com/2489570/43002579-4d278bc4-8bf7-11e8-924b-d383af0e3709.png)

![screenshot from 2018-07-20 08 29 57](https://user-images.githubusercontent.com/2489570/43002585-5015fa6e-8bf7-11e8-9f05-c18575c3d0b4.png)

![screenshot from 2018-07-20 08 30 24](https://user-images.githubusercontent.com/2489570/43002588-52ad53ee-8bf7-11e8-9b4a-28f38a07f4be.png)

There's a little bit of weirdness on one of the bulbs in the lamp model still, and I'm not sure what the cause is yet.

![screenshot from 2018-07-20 08 30 49](https://user-images.githubusercontent.com/2489570/43002621-6dc306ec-8bf7-11e8-88e1-07708ecba2d5.png)

**Comment by apptaro** [comment]
@lasalvavida wow, very nice, I'm looking forward to the next release. I have many other cases with this issue, so once the fix is released, I will try them all.

**Comment by apptaro** [comment]
any plan to release the fix?

**Comment by lasalvavida** [comment]
Hey @apptaro. It will be released once I have tested it more thoroughly.

If you want to use it/help test it in the meantime, you can grab pre-built artifacts for windows off of the appveyor job for the pull request like this one: [Windows x64 Release multiple-maps](https://ci.appveyor.com/api/buildjobs/ejwyif5levocy5cm/artifacts/cmake-build%2FCOLLADA2GLTF-master-windows-Release-x64.zip)

**Comment by apptaro** [comment]
@lasalvavida Yes, I would like to use and test it. I downloaded the one above, but it has the issue #200 and was not usable.

**Comment by apptaro** [comment]
@lasalvavida Let me know when I can test it out.

**Comment by lasalvavida** [comment]
Thanks @apptaro - the testing is very much appreciated!

**Comment by mycleaerlife** [comment]
I'm very looking forward that,but this is my stuff for multiple-maps and let that support THREE.js

**Comment by apptaro** [comment]
@lasalvavida After testing all my models, only cases with this issue are dress-pendant-light-l-black.dae (above) and dress-pendant-light-l-white.dae (attached here [data2.zip](https://github.com/KhronosGroup/COLLADA2GLTF/files/2316912/data2.zip))


Tested with Build 2.0.314, COLLADA2GLTF-master-windows-Release-x64.zip

**Comment by mycleaerlife** [comment]
![image](https://user-images.githubusercontent.com/35730929/44646545-f6fef180-aa0d-11e8-8248-72ca6c6fc8e7.png)
@lasalvavida it'my model .lightmap and diffusemap ,SO I have two layers of texture uvs for that and i I have modifieded the struct of outgltf and made it satisfied with it. and  I want to compile your program so that i can control what i can output.can you give me a corrent program on windows? Waiting online,（i can not solve the erro about camke's Compiling）

**Comment by lasalvavida** [comment]
@mycleaerlife, you can download Windows builds for any commit from the corresponding AppVeyor job artifacts tab like [this one](https://ci.appveyor.com/project/Khronoswebmaster/collada2gltf/build/2.0.313/job/n9nu1khnfi2ildmb/artifacts).

**Comment by mycleaerlife** [comment]
@lasalvavida I‘m sorry for that's wrong about my test, can i get source code ,and Compling it by myself.and sorry for my english 
![image](https://user-images.githubusercontent.com/35730929/44694493-45f56700-aaa0-11e8-91fb-2792c79b0e34.png)
